zilliqa/src/consensus.rs

```diff
@@ -1921,40 +1916,18 @@ impl Consensus {
         Ok(())
     }
 
-    pub fn leader_at_block(&self, block: &Block, view: u64) -> Option<Validator> {
+    pub fn leader_at_block(&self, block: &Block) -> Option<Validator> {
```

Comment:
As suggested above, we should use the `leaderAtView()` function of the `Deposit` contract with the `view` parameter.